packages/vuetify/types/cypress.d.ts

```diff
@@ -0,0 +1,9 @@
+import type { mount as cyMount } from '@cypress/vue'
+
+declare global {
+  namespace Cypress {
+    export interface Chainable {
+      mount: typeof cyMount
```

Comment:
This does not include `vuetifyOptions` argument

Reply:
I don't really know how to type it. I'll try agian.

Reply:
Since we're not actually using the argument yet I think we can revisit it later.